data.sql

```diff
@@ -18,4 +18,41 @@ SELECT * FROM animals;
  (10, 'Blossom', '1998-10-13', 3, 'true', 17),
  (11, 'Ditto', '2022-5-14', 4, 'true', 22);
 
- SELECT * FROM animals;
\ No newline at end of file
+ SELECT * FROM animals;
+
+   insert into vets(name, age, date_of_graduation)
+  values
+    ('William Tatcher', 45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-02-11');
```

Comment:
- Kindly, insert the correct data `Jack Harkness` birth date is `2008-06-08` so update it to follow the requirement 👍